app/views/content/subjects/maths/_article.html.erb

```diff
@@ -1,185 +1,160 @@
-        equations. It is a way of thinking, and problem solving, that’s important
-        in every walk of life.
+        Maths is critical to science, technology and engineering as well as being fundamental for most jobs.
       </p>
 
       <p>
-        An exciting career spent passing your passion on to the next generation
-        could be the career for you.
+        It is our government's goal for all young people to study maths to age 18, equipping them with the skills they need for life.
+      </p>
+      <p>
+        A good understanding of maths offers big benefits for young people as well as helping towards a strong economy.
       </p>
     <% end %>
   </section>
 
-  <section class="col col-640 col-space-s">
-    <%= render Content::QuoteComponent.new(
-      text: "You can literally change young people’s lives by being a maths teacher.
-          Be the person who helps them see the beauty and importance of maths.
-          It is the best job ever!",
-      name: "Kevin, maths teacher in Sheffield",
-      background: "white",
-      large: true
-    ) %>
-  </section>
-
   <section class="col col-720 col-horizontal col-space-m">
     <div>
-      <h2 class="heading-s">What you'll be teaching</h2>
-
-      <p>
-        In England, there are important aspects of maths to teach each age
-        group. As a secondary school teacher, maths education means teaching
-        pupils:
-      </p>
-    </div>
+      <h2>What you'll be teaching</h2>
+      <p>You'll teach the content of the national curriculum. The themes you are likely to cover include: </p>
+      <p>11 to 14 year olds (key stage 3):</p>
+      <ul class="vertical-tags">
+        <li class="tag">Number</li>
+        <li class="tag">Algebra</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
+        <li class="tag">Geometry and measures</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
 
-    <div>
+      <p>14 to 16 year olds (key stage 4):</p>
       <ul class="vertical-tags">
-        <li class="tag">Numbers</li>
+        <li class="tag">Number</li>
         <li class="tag">Algebra</li>
-        <li class="tag">Ratio, portion and rates of change</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
         <li class="tag">Geometry and measures</li>
-        <li class="tag">Probability</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
+
+      <p>16 to 18 year olds (key stage 5) - A or AS level maths:</p>
+      <ul class="vertical-tags">
+        <li class="tag">Proof</li>
+        <li class="tag">Algebra and functions</li>
+        <li class="tag">Coordinate geometry</li>
+        <li class="tag">Sequences and series</li>
+        <li class="tag">Trigonometry</li>
+        <li class="tag">Exponentials and logarithms</li>
+        <li class="tag">Differentiation</li>
+        <li class="tag">Integration</li>
+        <li class="tag">Numerical methods</li>
```

Comment:
this is quite a long list of topics for a levels - do we want to cut it down at all? do we need to mention them all?